neo/Consensus/ChangeView.cs

```diff
@@ -32,6 +33,8 @@ public override void Deserialize(BinaryReader reader)
             base.Deserialize(reader);
             Timestamp = reader.ReadUInt64();
             Reason = (ChangeViewReason)reader.ReadByte();
+            if (!Enum.IsDefined(typeof(ChangeViewReason), Reason))
```

Comment:
We don't need to check it here. If we add new reason in the future, it can be compatible with the current version.

Reply:
it's been a very long time that i looked at the consensus code, but what prevents me from sending a `ConsensusPayload` with an invalid message that gets processed unnecessarily (see https://github.com/neo-project/neo/pull/1254#discussion_r349537663 what I mean with unnecessary processing)?

Reply:
The consensus message doesn't need to be dropped even if I don't know the changeview reason. So this check is unnecessary because we may add new reasons in the future.